src/index.html

```diff
@@ -11,12 +12,103 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+  <!-- #endregion HEAD-->
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="index.html">
+          <img
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#apple"
+              class="nav_link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#samsung"
+              class="nav_link"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#smartphones"
+              class="nav_link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#laptops-and-computers"
+              class="nav_link"
+              data-qa="hover"
```

Comment:
The 'data-qa' attribute is used on the 'laptops & computers' link. Ensure this attribute is necessary and used correctly, typically for testing purposes.